indico/modules/categories/serialize.py

```diff
@@ -31,21 +31,23 @@
 from indico.web.flask.util import url_for
 
 
-def serialize_category_ical(category, user, event_filter):
+def serialize_category_ical(category_ids, user, event_filter=True, event_filter_fn=None, update_query=None):
```

Comment:
Since we are now accepting several IDs the method should reflect, somehow, this multiplicity. What about `serialize_categories_ical`?